buildSrc/src/main/java/app/tivi/buildsrc/dependencies.kt

```diff
@@ -172,20 +170,32 @@ object Libs {
 
             const val runtime = "androidx.compose:compose-runtime:$version"
         }
+
+        object Hilt {
+            private const val version = "1.0.0-alpha01"
+            const val work = "androidx.hilt:hilt-work:$version"
+            const val viewmodel = "androidx.hilt:hilt-lifecycle-viewmodel:$version"
+            const val compiler = "androidx.hilt:hilt-compiler:$version"
+        }
     }
 
     object Dagger {
         private const val version = "2.28"
```

Comment:
Dagger 2.28 comes with a new artifact for lint checks: `com.google.dagger:dagger-lint`

Reply:
Thanks, I'll add that in a later PR!